pages/dkp/konvoy/2.1/image-builder/create-azure-image/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: Create Azure Image
+title: Create a custom Azure Image
+excerpt: Learn how to build a custom Azure Image for use with Konvoy
+beta: false
+enterprise: false
+menuWeight: 60
+---
+
+This procedure describes how to use the Konvoy Image Builder to create a [Cluster API](https://cluster-api.sigs.k8s.io/) compliant Azure Virtual Machine (VM) Image. The VM Image contains the base operating system you specify and all the necessary Kubernetes components. The Konvoy Image Builder uses variable [`overrides`][overrides] to specify the base image and container images to use in your new Azure VM image.
+
+## Prerequisites
+
+Before you begin, you must:
+
+-   Download the latest [Konvoy Image Builder](https://github.com/mesosphere/konvoy-image-builder/releases) bundle (prefixed with `konvoy-image-bundle`) for your OS. Do not use the release prefixed with `konvoy-image-builder`.
+
+-   Create a working `Docker` setup.
+
+## Extract the bundle
+
+Extract the bundle and `cd` into the extracted `konvoy-image-bundle-$VERSION_$OS` folder. The bundled version of `konvoy-image` contains an embedded `docker` image that contains all the requirements for building.
+
+<p class="message--note"><strong>NOTE: </strong> The <code>konvoy-image</code> binary and all supporting folders are also extracted. When run, <code>konvoy-image</code> bind mounts the current working directory (<code>${PWD}</code>) into the container to be used.</p>
```

Comment:
@mwrightd2 
I don't know if Azure is valid to get built on 2.1.1.

I know it previously wasn't able to, but maybe the upgrades in KIB independently can make it work, but we may have to axe this.

What do you think @faiq ?

Reply:
I think its OK to leave it in and specify we need to use a v1.12.0 of KIB to build the images